ADA/databricks_rstudio_sql_warehouse.qmd

```diff
@@ -1,105 +1,191 @@
 ---
-  title: "Setup Databricks SQL Warehouse with RStudio"
+  title: "Using a SQL Warehouse to access Databricks data from RStudio"
 ---
 
-<p class="text-muted">The following instructions set up an ODBC connection between your laptop and your DataBricks SQL warehouse, which can then be used in R/RStudio to query data using an ODBC based package.
-SQL Warehouses are able to run SQL and can be used within the DataBricks environment, or through RStudio to run SQL code. The `sparklyr` package can also be used from within RStudio (only) as it converts the request for data to Spark SQL behind the scenes.</p>
+The following instructions will help you to set up a connection between your laptop and your Databricks SQL warehouse or personal cluster, which can then be used in RStudio to query data.
```

Comment:
Can probably remove "or personal cluster" here as these instructions no longer pertain to personal clusters.

Could always put a link to the instructions for a personal cluster below.